spec.html

```diff
@@ -553,7 +556,8 @@ <h1><ins>GetCycleRoot ( _module_ )</ins></h1>
       <h1><ins>CyclicModuleExecutionFulfilled ( _module_ )</ins></h1>
       <emu-alg>
         1. Assert: _module_.[[EvaluationError]] is *undefined*.
-        1. If _module_.[[Status]] is `"instantiating"`, then
+        1. Assert: _module_.[[Status]] is `"evaluating"`, `"evaluating-async"` or `"evaluated"`.
+        1. If _module_.[[Status]] is `"evaluating"`, then
           1. Assert: _module_.[[Async]] is *false*.
```

Comment:
It appears to be the case that this assertion now fails with the case of two mutually dependent async modules, with the stack Evaluate() -> InnerModuleEvaluation() -> InnerModuleEvaluation() -> ExecuteCyclicModule() -> CyclicModuleExecutionFulfilled().

Reply:
Thanks again for the excellent analysis. I've inlined the behaviour in the initial sync case to avoid these disparities.